examples/kuadrant/weighted-loadbalanced-healthchecks.yaml

```diff
@@ -1,48 +1,56 @@
 apiVersion: kuadrant.io/v1alpha1
 kind: DNSRecord
 metadata:
-  name: loadbalanced-healthchecks-dns
+  name: weighted-loadbalanced-healthchecks-dns
```

Comment:
not clear why all these have changed is it just a different subdomain or something more structural

Reply:
I was using "loadbalanced" for what was really "geo-loadbalanced" and now I also want to have "weighted-loadbalanced" so I renamed them to keep them unique.